docs/recipes/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "scripts": {
+    "install-all": "yarn --cwd auth/ && yarn --cwd integration-test/ && yarn --cwd custom-gatherer-puppeteer/",
+    "integration-test": "yarn --cwd integration-test/ jest --config-path=docs-jest.config.js",
+    "custom-gatherer-puppeteer-test": "yarn --cwd custom-gatherer-puppeteer/ test",
+    "test": "yarn install-all && yarn integration-test && yarn custom-gatherer-puppeteer-test"
```

Comment:
a little weird to run `install-all` under `test`, but since the only reason you'd ever run `install-all` (as opposed to installing just the single recipe you're looking at) is to run the tests, it seems fine 🤷